libraries/AP_Logger/LogFile.cpp

```diff
@@ -1042,13 +1042,14 @@ void AP_Logger::Write_SRTL(bool active, uint16_t num_points, uint16_t max_points
     WriteBlock(&pkt_srtl, sizeof(pkt_srtl));
 }
 
-void AP_Logger::Write_OABendyRuler(bool active, float target_yaw, float margin, const Location &final_dest, const Location &oa_dest)
+void AP_Logger::Write_OABendyRuler(bool active, float target_yaw, bool avoid_change, float margin, const Location &final_dest, const Location &oa_dest)
 {
     const struct log_OABendyRuler pkt{
         LOG_PACKET_HEADER_INIT(LOG_OA_BENDYRULER_MSG),
         time_us     : AP_HAL::micros64(),
         active      : active,
         target_yaw  : (uint16_t)wrap_360(target_yaw),
+        avoid_change: avoid_change,
```

Comment:
sorry to be so picky but could we move this new bool to the end?  If possible I like to keep the "desired" and "actual" columns next to each other.  I agree we should add this new value because it will help us with support but I don't think we will need to check it that often... so moving it to the end should be fine.

Reply:
.. or maybe instead of moving it to the end, just move it after the next "Yaw" column.  Then all the angle related values are together but the desired and actual yaw values are together.